docs/app/pages/css/css-sections/icons/icon-documentation/snippets/service.ts

```diff
@@ -0,0 +1,17 @@
+import { IconService } from '@ux-aspects/ux-aspects';
+
+export class AppComponent {
+
+    constructor(iconService: IconService) {
+
+        // replace single icon
+        iconService.setIcon({ name: '3d', icon: 'ux-icon-3d', iconset: 'ux-icons', size: '14px' });
```

Comment:
Perhaps these examples should use a different iconset, eg, quantum icons (http://quantum-ux.swinfra.net:8011/#!/components/icons) as currently this isn't really overriding anything?

Reply:
Changed to use quantum icons in examples